apps/sms/style/sms.css

```diff
@@ -37,6 +37,11 @@ gaia-header {
   z-index: 10;
 }
 
+gaia-header .group-header-title {
+  display: flex;
```

Comment:
Looks like with flex layout ```bdi``` with contact name and ```bdi``` with number of recipients stick together and there's no any space at all between them.